deployments/deploy-ssh.sh

```diff
@@ -88,6 +88,10 @@ if [ -n "${K8S_VENDOR}" ]; then
         rancher)
             K8S_VENDOR_KEY=node-role.kubernetes.io/controlplane
             ;;
+        gke)
+            # Run torpedo on worker node, where px installation is disabled. 
+            K8S_VENDOR_KEY=px/enabled
```

Comment:
actually `px/enabled` needs to be equal to `false` instead of just existing.

Add `K8S_VENDOR_VALUE` and keep it empty for others. Also update nodeSelectorTerms below

Reply:
Hi @harsh-px,  Updated PR to use In operator instead of Exists.